packages/riipen-ui/src/components/Drawer.test.jsx

```diff
@@ -0,0 +1,177 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import Drawer from "./Drawer.jsx";
+
+describe("<Drawer>", () => {
+  it("renders without errors", () => {
+    let error;
+    try {
+      mount(<Drawer />);
+    } catch (e) {
+      error = e;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  it("snapshot test when open is true and onClose is provided", () => {
```

Comment:
"it snapshot test when open is true and onClose is provided" is kind of a weird sentence to read.

Maybe we can do "it renders the correct snapshot when ...".